File: OrderConfirmation.vb
Public Class OrderConfirmation

    Public Property ordernumber As String

    Private Const CS_DROPSHADOW As Integer = 131072

    Protected Overrides ReadOnly Property CreateParams() As System.Windows.Forms.CreateParams

        Get
            Dim cp As CreateParams = MyBase.CreateParams
            cp.ClassStyle = cp.ClassStyle Or CS_DROPSHADOW
            Return cp
        End Get

    End Property

    Private Sub OrderConfirmation_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        lblOrderNum.Text = "Order number: #" & ordernumber
    End Sub

    Private Sub btnDone_Click(sender As Object, e As EventArgs) Handles btnDone.Click
        Me.DialogResult = DialogResult.OK
        Me.Close()
    End Sub
End Class

File: OrderConfirmation.cs
using System;
using System.Windows.Forms;

namespace MilkTeaOrderAndInventorySystem
{
    public partial class OrderConfirmation
    {

        public string ordernumber { get; set; }

        private const int CS_DROPSHADOW = 131072;

        protected override CreateParams CreateParams
        {

            get
            {
                var cp = base.CreateParams;
                cp.ClassStyle = cp.ClassStyle | CS_DROPSHADOW;
                return cp;
            }

        }

        public OrderConfirmation()
        {
            InitializeComponent();
        }

        private void OrderConfirmation_Load(object sender, EventArgs e)
        {
            lblOrderNum.Text = "Order number: #" + ordernumber;
        }

        private void btnDone_Click(object sender, EventArgs e)
        {
            DialogResult = DialogResult.OK;
            Close();
        }
    }
}